shared/product.go

```diff
@@ -111,3 +112,26 @@ func CertRotate(product string, ips []string) (string, error) {
 
 	return "", nil
 }
+
```

Comment:
honestly i think `SystemCtlCmd` , `ManageService` ,  `CertRotate`, and now `SecretEncryptOps` does not makes much sense here , since here should be really related to product functions itself

if we go think like, " oh but they use a product or something like that " then almost half of the code should live here lol

so i would really suggest that those go to cluster file

Reply:
so the commands that go :
sudo k3s --help 
sudo rke2 --help

they all belong here. They are testing k3s/rke2 product commands. 
the cluster ones are all kubectl operations.

Reply:
agree, my point was like , they are related to product only because of how we deal stuff in code and cause we have shared functions to rke2/k3s ..... but they are actually stuff/functionalities/aux  of the cluster itself 

got it ?

Reply:
like if i would do it now i would put manageservice AND systemctlcmd in AUX.go, AND  rotate and encrypt in cluster

makes sense or no ?

Reply:
aux.go is where we put auxilliary helper stuff. like logging or basic skeleton stuff. putting manageservice and systemcmdctl here - is like stuffing something somewhere cos we dont have a better place to put it. 
and rotate and encrypt is not cluster kubectl ops. i feel it makes sense to give purpose and value to product and k3s/rke2 commands themselves - cos "that is what we are testing". So it makes sense to give importance to what we are testing as products and give them the attention they deserve and keep them at product.go
End of the day, it boils down to perspectives. Its all "shared.x" anyway. 
Either way, it doesnt make sense to change this in this PR. If you still disagree, we should circle back on design for another discussion with team and keep it as a separate PR for a design change.